oqum/hazard/00a_sources_distributed_seismicity.tex

```diff
@@ -186,19 +158,15 @@ \subsubsection{Area sources}
 
 \end{itemize}
 
-Below we provide the exerpt of an .xml file used to describe the properties of
-an area source:
+Below we provide the excerpt of an .xml file used to describe the properties of an area source. The ruptures generated by the area source described in the example are controlled by two nodal planes and have hypocenters at localized at two distinct depths.
```

Comment:
It's a pity we lost the possibility of showing various components with different colours.